src/theme/css/chrome.css

```diff
@@ -427,8 +427,7 @@ ul#searchresults span.teaser em {
     -ms-user-select: none;
     user-select: none;
 }
-.no-js .sidebar,
-.js:not(.sidebar-resizing) .sidebar {
+html:not(.sidebar-resizing) .sidebar {
     transition: transform 0.3s; /* Animation: slide away */
```

Comment:
The `sidebar-resizing` and `sidebar-visible` classes are currently set on the body element. I would probably move them to the html element, just like all the other utility classes, so that you don't have to memorize which is which.

Right now, you get a very strange rubber-band effect when resizing the sidebar.

Reply:
Good point, updated it.